Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1168,6 +1168,21 @@ public func gelu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
     return x * cdf
 }
 
+/// Returns a tensor by applying the ReLU activation function to the specified tensor element-wise.
+/// Specifically, computes `max(0, x)`.
+@inlinable
+@differentiable(vjp: _vjpRelu(_:))
+public func relu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    Raw.relu(features: x)
+}
+
+@inlinable
+func _vjpRelu<T: TensorFlowFloatingPoint>(
+    _ x: Tensor<T>
+) -> (Tensor<T>, (Tensor<T>) -> Tensor<T>) {
+    (relu(x), { v in Tensor(x .> 0) * v })
```

Comment:
I'm not sure about the performance issues, but @eaplatanios there is a `reluGrad` available as well, What do you think about using that instead?

Reply:
Good point! I switched to using that.